elements/pfelement/src/pfelement.js

```diff
@@ -176,8 +207,8 @@ class PFElement extends HTMLElement {
     this.classList.add("PFElement");
     this.setAttribute("pfelement", "");
 
-    if (typeof this.props === "object") {
-      this._mapSchemaToProperties(this.tag, this.props);
+    if (typeof this.schemaProps === "object") {
```

Comment:
Should we add a TODO to deprecate this for 1.0?

Reply:
As in, remove `_mapSchemaToProperties`?

Reply:
Cassondra to handle this - need to update the storybook utils